src/cpu/flags.cpp

```diff
@@ -1053,8 +1053,18 @@ void set_cpu_test_flags_for_division(const T quotient) noexcept
 {
 	// Set the instruction type to division
 	lflags.type = t_DIV;
-	
+
+	// After performing IDIV, Intel's micro-operation clears the carry and
+	// overflow flags. Curiously, the 8086 documentation declares that the
+	// status flags are undefined following IDIV, but the microcode
+	// explicitly clears the carry and overflow flags. Signed DIV re-uses
+	// the above IDIV routine, so this is true for both signed and unsigned
+	// division.
+	//
+	//_https://www.righto.com/2023/04/reverse-engineering-8086-divide-microcode.html
+
 	const CpuTestFlags cpu_test_flags = {
+	        .HasCarry    = false,
 	        .HasOverflow = false,
```

Comment:
Same comment; I think where we use designated initialisers, we can't to provide all values otherwise we'll get a warning.

Reply:
Replaced with a slightly different approach.